### PR TITLE
MAT-6313 Feature flag for Execution of test cases containing observat…

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
…ions or stratifications

## MADiE PR

Jira Ticket: [MAT-6313](https://jira.cms.gov/browse/MAT-6313)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is in to the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     "webpack-config-single-spa-ts": "^4.1.2",
     "webpack-dev-server": "^4.9.3",
     "webpack-merge": "^5.8.0",
-    "webpack-pwa-manifest": "^4.3.0"
```
Comment (nmorasb) [inline package.json:55]: was this an intentional downgrade?

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
...
+    "webpack-pwa-manifest": "^4.0.0"
```
Comment (nmorasb) [inline package.json]: Was this an intentional downgrade? Was it causing some issue?

Comment (sb-cecilialiu) [reply package.json]: @nmorasb This one was due to the npm audit fix, should I revert it?

Comment (sb-cecilialiu) [reply package.json]: @nmorasb I reverted the package files, thanks!

**Review by nmorasb — APPROVED** [review]
